nest-forms-backend/src/nases-consumer/nases-consumer.service.ts

```diff
@@ -217,6 +219,7 @@ export default class NasesConsumerService {
     await this.formsService.updateForm(data.formId, {
       state: FormState.DELIVERED_NASES,
       error: FormError.NONE,
+      ...additionalFormUpdates,
```

Comment:
I really dislike like these spreads which ultimately end up in database query, but that's a problem of updateForm and it's sub-functions, not of this. So ok